sw/device/silicon_creator/lib/drivers/otbn.c

```diff
@@ -139,6 +139,10 @@ void otbn_get_err_bits(otbn_err_bits_t *err_bits) {
   *err_bits = abs_mmio_read32(kBase + OTBN_ERR_BITS_REG_OFFSET);
 }
 
+uint32_t otbn_get_instruction_count(void) {
+  return abs_mmio_read32(kBase + OTBN_INSN_CNT_REG_OFFSET);
+}
```

Comment:
We should also have `otbn_instruction_count_reset()` and call it before running the OTBN program since we can run it multiple times due to a/b slots and multiple boot stages.

Reply:
The hardware actually takes care of this -- the instruction counter automatically resets to zero when you execute a new program.

Reply:
I actually checked the HWIP spec and couldn't see it stated clearly but I guess the following covers this:
> Returns the number of instructions executed in the current or last operation. 
Thanks for the clarification. Can you add a short note to the comment of this function to avoid confusion in the future?